_config.yml

```diff
@@ -37,6 +37,11 @@ release_info:
     build_time: 2017/10/26 14:48:26 (go1.9)
     start_time: 2017-10-26 15:10:52.34274101 +0000 UTC
 
+training:
```

Comment:
Is it ok to put these details here, out in the open?

Reply:
Yeah, I've worked with Marc on this. They'll be temporary and we'll have to put them in and pull them out before/after the training.